internal/api/verify.go

```diff
@@ -415,8 +415,10 @@ func (a *API) prepErrorRedirectURL(err *HTTPError, w http.ResponseWriter, r *htt
 	q.Set("error_code", strconv.Itoa(err.Code))
 	q.Set("error_description", err.Message)
 	if flowType == models.PKCEFlow {
+		// Additionally, may override existing error query param if set to PKCE.
 		u.RawQuery = q.Encode()
 	}
+	// Left as hash fragment to comply with spec.
```

Comment:
My bad here - think the original comment could have been clearer. I think this only applies when we're doing an OAuth sign in. Since `prepErrorRedirectURL` is used in `verify.go` as well we can probably do without the comments here and make the original comment in `external.go` clearer
```suggestion
```